packages/eas-cli/src/submit/ios/CredentialsServiceSource.ts

```diff
@@ -16,15 +20,29 @@ import { AscApiKeyResult } from './AscApiKeySource';
  * The Credentials Service will either return an ASC API Key or an App Specific Password
  * When we no longer support the App Specific Password user prompt, refactor this into the AscApiKeySource
  */
-export const CREDENTIALS_SERVICE_SOURCE = 'CREDENTIALS_SERVICE_SOURCE';
-export type CredentialsServiceSource = typeof CREDENTIALS_SERVICE_SOURCE;
+export const CREDENTIALS_SERVICE_SOURCE = 'CREDENTIALS_SERVICE_SOURCE' as const;
+export interface CredentialsServiceSource {
+  sourceType: typeof CREDENTIALS_SERVICE_SOURCE;
+  bundleIdentifier?: string;
+}
+
+async function promptBundleIdentifierAsync(): Promise<string> {
+  const { bundleIdentifier } = await promptAsync({
+    name: 'bundleIdentifier',
+    message: 'Xcode bundle identifier:',
```

Comment:
```suggestion
    message: 'iOS bundle identifier:',
```
or just
```suggestion
    message: 'Bundle identifier:',
```
`Xcode` sounds weird.